### PR TITLE
Dependency updates 20230104

### PR DESCRIPTION
**Pull request by vanvoljg**

### CODE REVIEW

**Review by dhedlund — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Cocktail ![Cocktail](./logo_with_border.png)
```
Comment (doughsay) [inline README.md:1]: 😭

File: mix.exs [hunk 2]
```diff
@@ -39,7 +39,7 @@ defmodule Cocktail.Mixfile do
   defp package do
     [
       files: ["lib", "mix.exs", "README.md", "LICENSE.md"],
-      maintainers: ["Chris Dosé"],
+      maintainers: ["Jesse Van Volkinburg", "Chris Dosé"],
```
Comment (doughsay) [inline mix.exs:42]: 🎉

**Review by doughsay — APPROVED** [review]
😄

File: README.md (hunk 1 above)
Comment (vanvoljg) [reply README.md:1]: I didn't take any time at all to check how to force import this into the assets folder. It breaks in the ExDoc compiled view, so I just took it out. Feel free to push a commit :-P

Comment (doughsay) [reply README.md:1]: lol, no, this is fine 😅

**Pull request merged**